README.md

```diff
@@ -43,69 +43,95 @@ remotes::install_github("mrc-ide/frogger", upgrade = FALSE)
+| netmigr            | demography.net_migration                 | Net migration by age group, sex and year                                                                                                                              |
+| asfr               | demography.fertility_ratio               | Ratio of number of live births in a year and the whole female population of childbearing age                                                                          |
+| births_sex_prop    | demography.births_sex_prop               | Proportion of male and female births each year                                                                                                                        |
+| incidinput         | incidence.rate                           | Incidence rate per year                                                                                                                                               |
+| incrr_sex          | incidence.sex_rate_ratio                 | HIV incidence rate ratio for female : male age 15-49 years                                                                                                            |
+| incrr_age          | incidence.age_rate_ratio                 | HIV incidence rate ratio by age (for each sex)                                                                                                                        |
+| cd4_initdist       | natural_history.cd4_initial_distribution | Distribution of infections by CD4 category upon infection                                                                                                             |
+| cd4_prog           | natural_history.cd4_progression          | Probability of progressing from 1 CD4 stage to the next by age and sex                                                                                                |
+| cd4_mort           | natural_history.cd4_mortality            | Probability of mortality by CD4 stage, age and sex                                                                                                                    |
+| scale_cd4_mort     | natural_history.scale_cd4_mortality      | If 1 then scale HIV related mortality (i.e. cd4_mortality) as a proportion of number of people with HIV and over the number with HIV and on ART at this disease stage |
+| art_mort           | art.mortality                            | Probability of mortality by treatment stage, CD4 stage, age and sex                                                                                                   |
+| artmx_timerr       | art.mortaility_rate_ratio                | HIV-related mortality rate ratios by treatment stage                                                                                                                  |
```

Comment:
The `time` component is important in that this modifies the ART mortality rate over time

Reply:
Can you suggest a name here? And a better description for this?

Reply:
`art_mort_time_rateratio` or `art_mort_time_rr`

Description: "ART mortality rate ratio by year for <12 months and >12 months on ART"